### PR TITLE
[14.0] [MIG] account_invoice_report_grouped_by_picking_sale_mrp: Migration to 14.0

### PR DESCRIPTION
**Pull request by ofonseca-pyming**
Standard migration to 14.0
I recovered the hook method like @chienandalu did here https://github.com/OCA/account-invoice-reporting/pull/190
Review please @ernestotejeda

### CODE REVIEW

**Comment by navarrorico** [comment]
LGTM :+1:

**Review by navarrorico — APPROVED** [review]
Functional review

**Comment by pedrobaeza** [comment]
/ocabot migration account_invoice_report_grouped_by_picking_sale_mrp
/ocabot merge nobump

File: account_invoice_report_grouped_by_picking/models/account_move.py [hunk 1]
```diff
@@ -38,12 +49,7 @@ def lines_grouped_by_picking(self):
             for move in line.move_line_ids:
                 key = (move.picking_id, line)
                 picking_dict.setdefault(key, 0)
-                qty = 0
-                if move.location_id.usage == "customer":
-                    qty = -move.quantity_done * sign
-                    has_returned_qty = True
```
Comment (carlosdauden) [inline account_invoice_report_grouped_by_picking/models/account_move.py:44]: Where is has_returned_qty after this change?

**Pull request merged**